specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+        "remainingEnrollmentsSpeechLength": {
+          "$ref": "#/definitions/RemainingEnrollmentsSpeechLength"
+        },
+        "audioLength": {
+          "type": "number",
+          "description": "This enrolment audio length in seconds.",
+          "example": 1.83
+        },
+        "audioSpeechLength": {
+          "type": "number",
+          "description": "This enrollment audio pure speech (which is the amount of audio after removing silence and non-speech segments) length in seconds.",
+          "example": 1.35
+        }
+      }
+    },
+    "IdentifyInfo": {
+      "description": "Identified speaker info",
+      "properties": {
+        "profileId": {
+          "type": "string",
+          "pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$",
+          "description": "ID of identified of profile. If no candidate is identified as the right speaker, the value is set to empty GUID."
+        },
+        "score": {
+          "type": "number",
+          "description": "A float number indicating the similarity between input audio and targeted voice print. This number must be between 0 and 1. A higher number means higher similarity."
+        }
+      }
+    },
+    "Error": {
+      "required": [
+        "error"
+      ],
+      "properties": {
+        "error": {
+          "properties": {
+            "code": {
+              "type": "string"
+            },
+            "message": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "NotFound": {
+      "description": "NotFound - Requested profile doesn't exist",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "Not Found",
+          "message": "Requested profile doesn't exist"
+        }
+      }
+    },
+    "Unauthorized": {
+      "description": "Request is not authorized. Make sure subscription key is included and valid.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "Unauthorized",
+          "message": "Request is not authorized. Make sure subscription key is included and valid."
+        }
+      }
+    },
+    "RateLimit": {
+      "description": "Rate limit is exceeded.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "RateLimit",
+          "message": "Rate limit is exceeded."
+        }
+      }
+    },
+    "ServerError": {
+      "description": "Internal Server Error.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "code": "InternalServerError",
+          "message": "Internal Server Error."
+        }
+      }
+    }
+  },
+  "paths": {
+    "/text-independent/profiles": {
+      "post": {
```

Comment:
@HeidiHanZhang 
All of the operations should have an ```operationId```. You can see some guidelines for naming ```operationId``` [here](https://github.com/Azure/azure-spectral-ruleset/blob/main/openapi-style-guidelines.md#operationid)

Reply:
done